src/liberty/multiStepInput.ts

```diff
@@ -0,0 +1,366 @@
+import { QuickPickItem, window, Disposable, QuickInputButton, QuickInput, ExtensionContext, QuickInputButtons } from 'vscode';
+import * as devCommands from "./devCommands";
+import { getProjectOptions } from '../definitions/starterOptions';
+import * as vscode from "vscode";
+import * as fs from "fs";
+
+export async function multiStepInput(context: ExtensionContext) {
```

Comment:
This class uses tabs for indents but `starterOptions.ts` uses 4 spaces for indents. To keep consistent let's keep using 4 spaces for indents. Some existing classes may need updating to keep consistent, offhand I noticed `libertyProject.ts`

Reply:
Suggestion to add some JSDoc style comments to explain new classes/functions